obp/ope/meta_slate.py

```diff
@@ -112,9 +119,15 @@ class SlateOffPolicyEvaluation:
 
     bandit_feedback: BanditFeedback
     ope_estimators: List[BaseSlateOffPolicyEstimator]
+    base_regression_model: Optional[SlateRegressionModel] = None
+    is_factorizable: bool = False
 
     def __post_init__(self) -> None:
         """Initialize class."""
+        self.n_rounds = self.bandit_feedback["n_rounds"]
+        self.len_list = int((self.bandit_feedback["slate_id"] == 0).sum())
+        self.n_unique_action = self.bandit_feedback["n_unique_action"]
```

Comment:
[ask]

Are these three variables necessary? (Test failed here)

Reply:
No, I will delete these attributes (I used these in the previous implementation and forgot to delete..). Thank you.